co3d/download_dataset.py

```diff
@@ -292,9 +63,17 @@ def _download_with_progress_bar(url: str, fname: str, filename: str):
     parser.add_argument(
```

Comment:
Hard to understand why do we have both `no_checksum_check ` and `checksum_check `? Can you just keep `no_checksum_check`?

Reply:
It is conventional to have `--feature` and `--no-feature` pair of args to switch flags. Users don’t need to remember the default values. I can remove if you feel strongly about it though.

Reply:
ok let's keep it, I have never seen this before though.